---
layout: project
title: Presentation
permalink: /
---


## Welcome on the official web page of the ANR project **PanQueSt**.  

PanQueSt is a collaborative research project, funded by the [french funding agency ANR](https://anr.fr/en/), within the call AAP PRC 2025, for the period January 2026 - December 2029.

<img src="images/logo_ANR.jpg" width="100"/>



### Context: Structural Variation in the pangenomic era

Structural variants (SVs) include large-scale genomic alterations such as insertions, deletions, duplications, inversions and translocations. They represent a critical class of genetic variation historically under-estimated due to the limitations of early sequencing technologies in detecting them compared to smaller mutations like single nucleotide polymorphisms (SNPs). Although less common than SNPs, recent advances in genome sequencing have revealed the substantial role SVs play in shaping genetic diversity, phenotypic traits, and disease susceptibility. At the same time, the acceleration of high-quality “telomere-to-telomere” genome assembly has shifted genomic analysis from relying on a single reference genome to a more comprehensive pangenome model. This approach captures the full spectrum of genomic variation within a species. By incorporating multiple genomes into a single graph representation, pangenome graphs overcome the inherent reference bias present in the linear reference-based approaches and allow the detection of variations that may be absent from the reference. However, pangenome graphs are complex and large scale data structures, and pangenome-based approaches are still in their early stages. We currently lack both perspective and adapted computational tools in order to fully leverage this novel paradigm, particularly in the context of SV characterization.

### Objectives

The PanQueSt project gathers complementary skills in order to develop innovative computational tools for detecting and characterizing SVs in pangenome graphs, overcoming current biases towards smaller variants or model organisms. Our approach will include novel algorithms for accurately detecting complex SV motifs in pangenome graphs and refining their structural details, significantly improving resolution and interpretation. We will also develop a comprehensive benchmark dataset to standardize SV evaluation, providing a valuable resource for the genomics community. Finally, we will propose methods to filter SVs that can be associated to a genetic metadata, addressing the challenge of downstream-analysis scalability and contributing to understanding the missing heritability issue. By offering scalable and user-friendly tools, the project will facilitate new discoveries and societal applications in agronomy, health, and biodiversity.

<img src="images/logo_ANR.jpg" width="100"/>
